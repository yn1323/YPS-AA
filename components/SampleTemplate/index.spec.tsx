import React from 'react'
import renderer from 'react-test-renderer'
import Component from '.'

test(Component.name, () => {
  const component = renderer.create(<Component />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})

// describe('Sample.ts Functions TestCases', () => {
//   it('should return the squared value', () => {
//     const result = square(2)
//     const expected = 4

//     expect(result).toStrictEqual(expected)
//   })
// })
