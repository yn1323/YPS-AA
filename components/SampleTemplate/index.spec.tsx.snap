// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component 1`] = `
<button>
  this is button
</button>
`;
